### PR TITLE
Update set-up-database in correspondance with the currrent configurat…

### PR DESCRIPTION
**Pull request by rguillome**
…ion code

In this code (https://github.com/apache/airflow/blob/5b31b63203391dd39f485f1c9013ea55c83ed1aa/airflow/configuration.py#L777) postgres+psycopg2 is a bad scheme

so the documentation shouldn't mention that as a recommandation almost with an ident that seems to be out of the warning about the old version of sql_alchemy

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/howto/set-up-database.rst [hunk 1]
```diff
@@ -198,12 +198,6 @@ in the Postgres documentation to learn more.
 
    Details in the `SQLAlchemy Changelog <https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html#change-3687655465c25a39b968b4f5f6e9170b>`_.
 
-We recommend using the ``psycopg2`` driver and specifying it in your SqlAlchemy connection string.
-
-.. code-block:: text
-
-   postgresql+psycopg2://<user>:<password>@<host>/<db>
```
Comment (Taragolis) [inline docs/apache-airflow/howto/set-up-database.rst:205]: > postgres+psycopg2 is a bad scheme

postgres**ql**+psycopg2 `!=` postgres+psycopg2

Comment (Taragolis) [reply docs/apache-airflow/howto/set-up-database.rst:205]: https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg2

Comment (potiuk) [reply docs/apache-airflow/howto/set-up-database.rst:205]: Yes. `postgres+psycopg2` used to be possible in sqlalchemy but it's been deprecated and removed, So our recommendation is correct.

**Comment by potiuk** [comment]
Closing - as this was misunderstanding/typo mistake.

**Pull request closed without merging**